### PR TITLE
Fix mwgs pipeline

### PR DESCRIPTION
**Pull request by ingkebil**
Revisit how to determine if a sample belongs to the `mwgs` pipeline.

### CODE REVIEW

File: cglims/api.py [hunk 1]
```diff
@@ -39,8 +39,12 @@ def pipeline(self):
 
         Returns (str): 'mip' or 'mwgs'
         """
-        if self.lims.udf.get('tissue_type') != 'tumour':
-            return 'mip' if self.apptag.is_human else 'mwgs'
+        if self.lims.get('tissue_type') != 'tumour':
```
Comment (robinandeer) [inline cglims/api.py]: uhmm.. the LIMS sample itself doesn't have this information, right? Is it a UDF?

File: cglims/api.py [hunk 2]
```diff
@@ -39,8 +39,12 @@ def pipeline(self):
...
+            if self.apptag.is_human:
+                return 'mip'
+
+        if self.apptag.is_microbial:
```
Comment (robinandeer) [inline cglims/api.py]: maybe `elif` makes it clearer what is happening here?

File: cglims/cli/commands.py [hunk 3]
```diff
@@ -119,7 +119,8 @@ def get(context, condense, project, external, identifier, field, all_samples):
         values['is_human'] = clinical_sample.apptag.is_human
         values['is_production'] = (False if values['customer'] == 'cust000'
                                    else True)
-        values['pipeline'] = clinical_sample.pipeline
+        pipeline = clinical_sample.pipeline
+        values['pipeline'] = pipeline if pipeline else 'NA'
```
Comment (robinandeer) [inline cglims/cli/commands.py]: why is `NA` preferred here over more Pythonic `None`?

File: cglims/api.py (hunk 1 above)
Comment (ingkebil) [inline cglims/api.py]: ... typo ;)

File: cglims/api.py (hunk 2 above)
Comment (ingkebil) [inline cglims/api.py]: hoh. I could change that, yes. I usually like how if statements are stand-alone when they are used for flow control. You know that each if ends in a `return` and I find it convoluted to add `elif` there.
But if it makes it clearer, of course!

File: cglims/cli/commands.py (hunk 3 above)
Comment (ingkebil) [inline cglims/cli/commands.py]: When it gets printed, one wants to see an `NA`, not a `null` or `None`. I tested it out with `None` as well, and it strangely resulted in `null` being printed. I presume this is the `click.echo` converting `None` to `null`?

**Comment by ingkebil** [comment]
OK, fixed the udf and flow control. I'll leave the `NA` for pipeline?

File: cglims/cli/commands.py (hunk 3 above)
Comment (robinandeer) [inline cglims/cli/commands.py]: I think null is the JSON/YAML representation
but then it makes sense!

**Pull request merged**